engine/access/rpc/backend/backend_accounts.go

```diff
@@ -41,6 +42,7 @@ func (b *backendAccounts) GetAccountAtLatestBlock(ctx context.Context, address f
 
 	account, err := b.getAccountAtBlockID(ctx, address, latestBlockID)
 	if err != nil {
+		b.log.Error().Err(err).Msg(fmt.Sprintf("failed to get account at blockID: %v", latestBlockID))
```

Comment:
Nit: zerolog has built in support for message formats
```suggestion
		b.log.Error().Err(err).Msgf("failed to get account at blockID: %v", latestBlockID)
```